myuw/static/vue/vuex/store/schedule/student.js

```diff
@@ -43,25 +47,7 @@ function postProcess(response, urlExtra) {
         }
       }
 
-      if (meeting.start_time && meeting.end_time) {
-        meeting.start_time = dayjs(meeting.start_time, "hh:mm")
-          .second(0)
-          .millisecond(0);
-        meeting.end_time = dayjs(meeting.end_time, "hh:mm")
-          .second(0)
-          .millisecond(0);
-      }
-
-      if (meeting.eos_start_date && meeting.eos_end_date) {
-        section.hasEosDates = true;
-        meeting.eos_start_date = dayjs(meeting.eos_start_date)
-          .second(0)
-          .millisecond(0);
-        meeting.eos_end_date = dayjs(meeting.eos_end_date)
-          .second(0)
-          .millisecond(0);
-      }
-
+      section.hasEosDates = meeting.eos_start_date && meeting.eos_end_date;
```

Comment:
If the last meeting has no start and end dates, section.hasEosDates will end up being false. 
This is different from setting "section.hasEosDates = true;" which will be true as long as one of the meetings has start and end date.